app/controllers/cards_controller.rb

```diff
@@ -42,7 +42,8 @@ def update
   private
 
   def card_params
-    params.require(:card).permit(:original_text, :translated_text, :review_date)
+    params.require(:card).permit(:original_text, :translated_text, 
+				 :review_date, :card_photo)
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.